lib/src/markdown.dart

```diff
@@ -296,3 +307,28 @@ class _MarkdownPageState extends State<MarkdownPage> {
     );
   }
 }
+
+/// Defines a handler that can intercept and handle taps on links
+/// in markdown templates. Implement this or extend [UrlMarkdownTapHandler].
+///
+abstract class MarkdownTapHandler {
```

Comment:
Why not just a callback?

Reply:
Because this way it can be extended and delegated to super for default behavior. I for example need to handle mostly real URLs but some are in-app navigation links. I only want to handle those myself and delegate all 'real' URLs to super.

Reply:
I see, thanks.